xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESink.h

```diff
@@ -40,6 +40,8 @@ struct SinkConfig
   AEAudioFormat format;
   CEngineStats *stats;
   const std::string *device;
+  int extSilenceTimeOut;
+  bool streamNoise;
```

Comment:
this does not fit well here, better send it with CSinkControlProtocol::STREAMING